fedimint-cli/src/lib.rs

```diff
@@ -696,8 +696,13 @@ impl FedimintCli {
                 peer_id,
                 password: auth,
             }) => {
-                let params: Value = serde_json::from_str(&params)
-                    .map_err_cli_msg(CliErrorKind::InvalidValue, "Invalid JSON-RPC parameters")?;
+                let params = serde_json::from_str::<Value>(&params).unwrap_or_else(|_| {
+                    serde_json::Value::String(
+                        // If parsing as JSON fails, treat it as a JSON string
+                        serde_json::to_string(&params).expect("Failed to convert to JSON String"),
+                    )
```

Comment:
incorrect?
`foo` will be converted to `Value::String("\"foo\"")`

Reply:
Actually ,It gives error when I try to return `Json_string` because of `mismatched types` .
I am not getting how to solve this problem ,
can you please help me out.

Reply:
```suggestion
                    serde_json::Value::String(params)
```

Reply:
Do I have to handle error related  `serde_json::Value::String(params)`?
if Yes, then should I return `Value::Null` in case of error ?